src/scheduler/buckets.cpp

```diff
@@ -911,9 +911,9 @@ int job_should_use_buckets(resource_resv *resresv) {
 
 	/* Job's requesting specific hosts or vnodes use the standard path */
 	const auto& defs = resresv->select->defs;
-	if (defs.find(getallres(RES_HOST)) != defs.end())
+	if (defs.find(allres["host"]) != defs.end())
```

Comment:
I actually like accessing the resource via this getter function that we previously had. If you want, you can just turns the enums to macros of the resource name and then you don't have to make this change in so many places.